.gitignore

```diff
@@ -4,6 +4,8 @@ npm-debug.log
 
 # Local development
 *.env
+*/react-ui/src/env.js
+*/server/env.js
```

Comment:
Ah, I see, yes. I have now created two commits: one removing the gitignore and the two files, and another restoring the gitignore commands.

Reply:
It looks like there are still several references to the now deleted .env file:

backend:
```
Error: Cannot find module './env'
5:58:49 AM web.1 |  Require stack:
5:58:49 AM web.1 |  - /workspaces/vexillology-contests/server/logger.js
```

frontend:
```
ERROR in ./src/common/FeatureBanner.jsx 19:0-32
Module not found: Error: Can't resolve '../env' in '/workspaces/vexillology-contests/react-ui/src/common'

ERROR in ./src/components/DevBar.jsx 22:0-59
Module not found: Error: Can't resolve '../env' in '/workspaces/vexillology-contests/react-ui/src/components'

ERROR in ./src/components/PageWithDrawer.jsx 25:0-38
Module not found: Error: Can't resolve '../env' in '/workspaces/vexillology-contests/react-ui/src/components'

ERROR in ./src/data/LocalStorageProvider.jsx 9:0-45
Module not found: Error: Can't resolve '../env' in '/workspaces/vexillology-contests/react-ui/src/data'

ERROR in ./src/pages/hallOfFame/HallOfFame.jsx 27:0-41
Module not found: Error: Can't resolve '../../env' in '/workspaces/vexillology-contests/react-ui/src/pages/hallOfFame'

ERROR in ./src/serviceWorker.js 16:0-31
Module not found: Error: Can't resolve './env' in '/workspaces/vexillology-contests/react-ui/src'
```

If the file is deleted from source control, it can't be referenced in these files since the build process on the server pulls from source control.

Reply:
Now, with the new approach, there is a webpack warning in the console. I added #307 to look into how to suppress that warning later, but it's not needed for this PR.